Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1048,7 +1048,7 @@ void NetworkConnectionToWebProcess::writeBlobsToTemporaryFilesForIndexedDB(const
             file->revokeFileAccess();
 
         if (auto* session = networkSession())
-            session->storageManager().registerTemporaryBlobFilePaths(m_connection, filePaths);
+            Ref { session->storageManager() }->registerTemporaryBlobFilePaths(Ref { m_connection }, filePaths);
```

Comment:
This looks a little strange.

Reply:
Oh yeah, I should be adding protectedStorageManager() here. Will fix.